experiments/interpretation/dutchf3_patch/local/train.py

```diff
@@ -161,6 +140,11 @@ def run(*options, cfg=None, debug=False):
     )
     logger.info(val_set)
 
+    if debug:
+        logger.info("Running in Debug Mode")
+        train_set = data.Subset(train_set, list(range(config.TRAIN.BATCH_SIZE_PER_GPU)))
```

Comment:
we should check that this works somehow... I guess the build VM will take care of this? Fix is https://github.com/microsoft/seismic-deeplearning/issues/238 so if it breaks replace with toolz.take again

Reply:
Yeah agreed. If this doesn't work, we should add this as a separate issue, and fix it in a future PR